packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -9,26 +9,82 @@ const setupNewUser = require('./setup-new-user');
 const obfuscateUtil = require('./utility-obfuscate');
 const systemConfigManager = require('./system-config-manager');
 
+const defaultAWSConfig = {
+  useProfile: true,
+  profileName: 'default',
+};
+
 async function init(context) {
-  context.projectConfigInfo = {};
+  normalizeInputParams(context);
+  context.exeInfo.awsConfigInfo = {
```

Comment:
Really hard to infer as to what the configLevel means.

Reply:
see above comments, the user can ignore it if he does not know about it, and it won't hurt him, by default it's on the project level,